liquibase/changeLogs/capture/indexes/jsonData.yml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - preConditions:
+    - dbms:
+        type: postgresql
+    - runningAs:
+        username: ${AQTS_SCHEMA_OWNER_USERNAME}
+
+  - changeSet:
+      author: kkehl
+      id: "create.index.${AQTS_SCHEMA_NAME}.json_data_uuid"
+      changes:
+        - sql: create index if not exists json_data_uuid on ${AQTS_SCHEMA_NAME}.json_data (partition_number, uuid);
+        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.json_data_uuid;
```

Comment:
I'm wondering if we should also have indices for each of the partition tables on uuid... :thinking:

Reply:
@ayan-usgs 
It looks like line #12 does that.  I'm looking in pgadmin, and I see indexes on the partitions, like:

Tables->json_data->Partitions->json_data_august->indexes->json_data_august_partition_number_uuid_idx

Reply:
Huh... I only saw the one on json_id. :man_shrugging: Alright then.

Reply:
Actually you are correct.  The way I had named it was confusing.  I have changed it so there are indexes on json_data_id and uuid on all partitions.